baseapp-payments/testproject/settings.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+
+from baseapp_core.settings.env import env
+from baseapp_core.tests.settings import *  # noqa
+
+# import from source code dir
+here = os.path.abspath(os.path.dirname(__file__))
+sys.path.insert(0, here)
+sys.path.insert(0, os.path.join(here, os.pardir))
+
+SITE_ID = 300
+
+DEBUG = True
+
+ROOT_URLCONF = "testproject.urls"
+SECRET_KEY = "very-secret"
+
+AUTOCOMMIT = True
+
+DATABASES = {
+    "default": {
+        "NAME": "test.db",
+        "ENGINE": "django.db.backends.sqlite3",
+        "USER": "",
+        "PASSWORD": "",
+        "PORT": "",
+    },
+}
+
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.contrib.auth.context_processors.auth",
+                "django.template.context_processors.request",
+                "django.contrib.messages.context_processors.messages",
+            ]
+        },
+    },
+]
+
+INSTALLED_APPS = [
+    # Django
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.sites",
+    "django.contrib.staticfiles",
+    # Third party
+    "djstripe",
+    "constance",
+    "constance.backends.database",
+    "baseapp_payments",
+]
```

Comment:
Let's reuse and complement INSTALLED_APPS imported from baseapp_core/tests/settings

```suggestion
INSTALLED_APPS += [
    # Django
    "django.contrib.sites",
    # Third party
    "djstripe",
    "constance",
    "constance.backends.database",
    "baseapp_payments",
]
```